src/tools/arakoon_logger.ml

```diff
@@ -71,30 +71,22 @@ let format_message
       ~ts ~seqnum
       ~section ~level ~lines
   =
+
+  let d =
+    let (f : float) = Obj.magic ts in
```

Comment:
do we need to use Obj.magic here?
I would expect there to be another way ...
(and if there isn't: can we specify the type of `ts` so callers can't pass in something of the wrong type?)